src/main/webapp/js/pages/index.js

```diff
@@ -34,10 +34,41 @@ document.querySelector('.course-list').addEventListener('click', () => {
       course => {sections2[course] = courseInfo[course].sections[1]});
   const schedules = [sections1, sections2];
 
-  // TODO(ramyabuva): add pagination for multiple schedules
-  selected.forEach(course => {
-    Calendar.addCourse(courseInfo[course], schedules[0][course]);
-  });
+  function addScheduleToCalendar(schedule){
+    Calendar.clear();
+    selected.forEach(course => {
+      Calendar.addCourse(courseInfo[course], schedule[course]);
+    });
+  }
+
+  addScheduleToCalendar(schedules[0]);
+
+  // if more than 1 schedule returned, create pagination
+  if (schedules.length == 1) {
+    return;
+  }
+  const pageList = document.getElementById('calendar-pagination');
+  pageList.innerText = ''; // clear any existing children from the element
+  for (let i = 1; i <= schedules.length && i <= MAX_PAGINATION_SCHEDULES; i++) {
+    const nextPage = document.createElement('li');
+    nextPage.innerText = i;
+    // by default, set first page to active
+    if (i == 1) {
+      nextPage.setAttribute('class', 'active');
+    }
+    pageList.appendChild(nextPage);
+    const NumSchedule = schedules[i - 1];
```

Comment:
`NumSchedule` -> `numSchedule`. variables declared in functions start with a lowercase. But also, is `schedule` a better name for this? Why is "num" relevant?